src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -5559,7 +5559,7 @@ namespace Google.Example.Library.V1
             Modify_DiscussBookRequestCallSettings(ref callSettings);
             BidirectionalStreamingSettings effectiveStreamingSettings =
                 streamingSettings ?? _callDiscussBook.StreamingSettings;
-            AsyncDuplexStreamingCall<DiscussBookRequest,Comment> call =
+            AsyncDuplexStreamingCall<DiscussBookRequest,  Comment> call =
```

Comment:
This seems to have gone from no spaces to two spaces... I suspect that wasn't deliberate...